### DISCUSSION TITLE
Portrait Image can't crop bottom part?

### DISCUSSION THREAD

**Discussion by ovidiuurotaru** [olutter/crop#70] [Q&A] [answered]
So I have a portrait image but I can't crop the bottom part (no rotation, no zoom) .
I basically want to just slide the image to the bottom  but. it pops to center always.

**Comment by xclud** [comment, marked as answer]
Hi @ovidiuurotaru 

This is a dup of #27.

**Reply by ovidiuurotaru** [reply]
Thx
